_data/projects/HoneyBot.yml

```diff
@@ -0,0 +1,10 @@
+name: HoneyBot
+desc: python powered OOP modular (plugins-supported) IRC bot. ignited in Mauritius.
+site: 
```

Comment:
@Abdur-rahmaanJ Please add link to your repo here

Reply:
ok